DigitalLearningSolutions.Web/Views/MyAccount/_EditRegistrationNumber.cshtml

```diff
@@ -0,0 +1,63 @@
+﻿@using DigitalLearningSolutions.Data.Enums
+@using DigitalLearningSolutions.Web.ViewModels.MyAccount
+@model EditDetailsViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid && @Model.ProfessionalRegNumberSelectionAnswer == YesNoSelectionEnum.None;
+  var optionYesSelected = @Model.ProfessionalRegNumberSelectionAnswer == YesNoSelectionEnum.Yes;
+  var optionNoSelected = @Model.ProfessionalRegNumberSelectionAnswer == YesNoSelectionEnum.No;
+  var professionalRegConditionalId = "professional-reg-conditional-Id";
+}
+
+<div class="nhsuk-form-group @(errorHasOccurred? "nhsuk-form-group--error": "")">
+  <fieldset class="nhsuk-fieldset" aria-describedby="professional-registration-hint">
+    <div class="nhsuk-hint" id="professional-registration-hint">
+      Do you have a Professional Registration Number?
+    </div>
+
+  @if (errorHasOccurred) {
+    <span class="nhsuk-error-message" id="no-selection-error">
+      <span class="nhsuk-u-visually-hidden">Error:</span> @(!optionYesSelected? "Select an option":"")

```

Comment:
We should be getting the error message from the error itself ideally, as this currently is if something unexpected happens we could display a message to users that just says "Error:"

Reply:
sorry, yes, I removed it as we don't need this here